plugins/woocommerce/src/Admin/Features/OnboardingTasks/Tasks/CustomizeStore.php

```diff
@@ -63,11 +73,114 @@ public function can_view() {
 	}
 
 	/**
-	 * Action URL.
-	 *
-	 * @return string
+	 * Possibly add site editor scripts.
 	 */
-	public function get_action_url() {
-		return admin_url( 'wp-admin/admin.php?page=wc-admin&path=%2Fcustomize-store' );
```

Comment:
Removed the `get_action_url` because we need to do a full redirect to load site editor scripts instead of using admin router.

Reply:
I think it's possible. I'll create a spike for this. I think we'll need to implement a RESTFul endpoint, get the block editor settings, and inject the wp scripts/styles dynamically.